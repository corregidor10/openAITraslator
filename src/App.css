.app{
    display: flex;
    flex-direction: column;
   justify-content: center;   
   align-items: center;
   width: 100%;
    height: auto;
    padding: 64px 0;
    background-color: white;
    row-gap: 32px;
   
}
.app-title{
font-size: 32px;
color: black;
opacity: 0.7;

}

