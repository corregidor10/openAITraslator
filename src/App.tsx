import { Traslator } from './components/traslator/Traslator';
import './App.css';

interface AppProps {
    title: string;
}
function App(props: AppProps) {
    return (
        <>
            <div className="app">
                <div className="app-title">{props.title}</div>
                <Traslator />
            </div>
        </>
    );
}

export default App;
