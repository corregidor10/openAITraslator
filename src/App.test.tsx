import { render } from '@testing-library/react';
import App from './App';

describe('App', () => {
    test('renders App component', () => {
        const { container } = render(<App title="Test" />);

        expect(container).toBeTruthy();
    });

    test('renders App component with title', () => {
        const { getByText, container } = render(<App title="Test" />);
        const titleElement = getByText('Test');

        expect(container.contains(titleElement));
    });
});
