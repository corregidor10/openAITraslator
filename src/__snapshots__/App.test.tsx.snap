// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`App > renders App component 1`] = `
<div>
  <div
    class="app"
  >
    <div
      class="traslator-container"
    >
      <button
        class="translator-button"
        disabled=""
      >
        Traducir
      </button>
      <div
        class="textarea-container"
      >
        <div
          class="textarea-item"
        >
          <div
            class="flagContainer"
          >
            <img
              class="flag"
              src="https://cdn.jsdelivr.net/gh/lipis/flag-icons/flags/4x3/gb.svg"
              style="display: inline-block; width: 1em; height: 1em; vertical-align: middle;"
              title="GB"
            />
            <img
              class="selectedFlag"
              src="https://cdn.jsdelivr.net/gh/lipis/flag-icons/flags/4x3/es.svg"
              style="display: inline-block; width: 1em; height: 1em; vertical-align: middle;"
              title="ES"
            />
            <img
              class="flag"
              src="https://cdn.jsdelivr.net/gh/lipis/flag-icons/flags/4x3/fr.svg"
              style="display: inline-block; width: 1em; height: 1em; vertical-align: middle;"
              title="FR"
            />
            <img
              class="flag"
              src="https://cdn.jsdelivr.net/gh/lipis/flag-icons/flags/4x3/de.svg"
              style="display: inline-block; width: 1em; height: 1em; vertical-align: middle;"
              title="DE"
            />
            <img
              class="flag"
              src="https://cdn.jsdelivr.net/gh/lipis/flag-icons/flags/4x3/it.svg"
              style="display: inline-block; width: 1em; height: 1em; vertical-align: middle;"
              title="IT"
            />
          </div>
          <textarea
            class="textarea"
          />
        </div>
        <div
          class="textarea-item"
        >
          <div
            class="flagContainer"
          >
            <img
              class="selectedFlag"
              src="https://cdn.jsdelivr.net/gh/lipis/flag-icons/flags/4x3/gb.svg"
              style="display: inline-block; width: 1em; height: 1em; vertical-align: middle;"
              title="GB"
            />
            <img
              class="flag"
              src="https://cdn.jsdelivr.net/gh/lipis/flag-icons/flags/4x3/es.svg"
              style="display: inline-block; width: 1em; height: 1em; vertical-align: middle;"
              title="ES"
            />
            <img
              class="flag"
              src="https://cdn.jsdelivr.net/gh/lipis/flag-icons/flags/4x3/fr.svg"
              style="display: inline-block; width: 1em; height: 1em; vertical-align: middle;"
              title="FR"
            />
            <img
              class="flag"
              src="https://cdn.jsdelivr.net/gh/lipis/flag-icons/flags/4x3/de.svg"
              style="display: inline-block; width: 1em; height: 1em; vertical-align: middle;"
              title="DE"
            />
            <img
              class="flag"
              src="https://cdn.jsdelivr.net/gh/lipis/flag-icons/flags/4x3/it.svg"
              style="display: inline-block; width: 1em; height: 1em; vertical-align: middle;"
              title="IT"
            />
          </div>
          <textarea
            class="textarea"
            disabled=""
          />
        </div>
      </div>
    </div>
    ;
  </div>
</div>
`;
